# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/reftests/svg/text/reftest.list
# Commit: 193184268c8e
# Full Hash: 193184268c8ef6347a77ef450d0ce008a7a360ee
# Author: longsonr <longsonr@gmail.com>
# Date: 2021-03-01 09:36:12
# Regressor Bug: 1600855
# File Overlap Count: 1
# Description:
#   Bug 1695490 - Remove svg.text-spacing.enabled pref r=emilio
#   
#   Backs out bug 1599173 which landed in Firefox 72. The pref has been enabled since bug 1600855 which landed in Firefox 73
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D106731
# ==============================================================================

diff -r 587d8a869c38 -r 193184268c8e layout/reftests/svg/text/reftest.list
--- a/layout/reftests/svg/text/reftest.list	Sun Feb 28 21:44:17 2021 +0000
+++ b/layout/reftests/svg/text/reftest.list	Sun Feb 28 21:52:21 2021 +0000
@@ -208,6 +208,6 @@
 fuzzy-if(skiaContent,0-1,0-200) needs-focus == textpath-selection.svg textpath-selection-ref.svg
 
 # letter-spacing and word-spacing
-pref(svg.text-spacing.enabled,true) == simple-letter-spacing.svg simple-letter-spacing-ref.svg
-pref(svg.text-spacing.enabled,true) == simple-word-spacing.svg simple-word-spacing-ref.svg
-pref(svg.text-spacing.enabled,true) == multiple-chunks-letter-spacing.svg multiple-chunks-letter-spacing-ref.svg
+== simple-letter-spacing.svg simple-letter-spacing-ref.svg
+== simple-word-spacing.svg simple-word-spacing-ref.svg
+== multiple-chunks-letter-spacing.svg multiple-chunks-letter-spacing-ref.svg